File: python/1047.py
a,c,b,d=list(map(int,input().split()))

dif=((b*60)+d)-((a*60)+c)
if(dif<=0):
    dif+=24*60
    
time=dif//60
minute=dif%60
print(f"O JOGO DUROU {time} HORA(S) E {minute} MINUTO(S)")

File: javascript/1047.js
const stdin = require("fs").readFileSync("/dev/stdin", "utf8");
const [initHour, initMin, endHour, endMin] = stdin.split(" ");

const print = (hours, minutes) =>
  console.log(`O JOGO DUROU ${hours} HORA(S) E ${minutes} MINUTO(S)`);

let totalHours = Number(endHour) - Number(initHour);
let totalMins = Number(endMin) - Number(initMin);

if (totalHours < 0) totalHours = 24 - Math.abs(totalHours);

if (totalMins < 0) {
  totalHours -= 1;
  totalMins = 60 - Math.abs(totalMins);
}

if (totalHours < 0) totalHours = 24 + totalHours;

if (totalHours === 0 && totalMins === 0) print(24, 0);
else print(totalHours, totalMins);